sql/sql_load.cc

```diff
@@ -418,6 +418,31 @@ int mysql_load(THD *thd, const sql_exchange *ex, TABLE_LIST *table_list,
   {
     DBUG_RETURN(TRUE);
   }
+
+#ifdef WITH_WSREP
+  /*
+    We support load data splitting for InnoDB only as it will use
+    streaming replication (SR).
+  */
+  if (WSREP(thd) && wsrep_load_data_splitting)
```

Comment:
Could this block be moved inside `Wsrep_load_data_split` constructor to keep the footprint in this function small?